tests/phpunit/tests/functions/_wpCheckExistingFileNames.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::wp_is_numeric_array
+ */
+class Tests_Functions__wp_check_existing_file_names extends WP_UnitTestCase {
```

Comment:
the class name should rename as per [doc](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes) i think